db/re/item_combo_db.txt

```diff
@@ -793,6 +793,58 @@
 19139:22170,{ bonus bVit,4; bonus bInt,5; bonus2 bSubRace,RC_DemiHuman,3; bonus2 bSubRace,RC_Player,3; }
 19172:28500,{ bonus bDelayrate,-15; }
 19172:4403,{ bonus bDelayrate,30; }
+19189:29316,{ autobonus3 "{ bonus bCritical,5; }",1000,60000,"BS_ADRENALINE"; }
+19189:29317,{ autobonus3 "{ bonus bCritical,10; }",1000,60000,"BS_ADRENALINE"; autobonus "{ bonus2 bAddSize,Size_All,10; }",20,5000,BF_WEAPON;  }
+19189:29318,{ autobonus3 "{ bonus bCritical,20; bonus bCritAtkRate,10; }",1000,60000,"BS_ADRENALINE"; autobonus "{ bonus2 bAddSize,Size_All,20; }",50,10000,BF_WEAPON;  }
+19190:29319,{ autobonus3 "{ bonus2 bAddSize,Size_All,5; }",1000,60000,"GN_CARTBOOST"; }
+19190:29320,{ autobonus3 "{ bonus2 bAddSize,Size_All,7; }",1000,60000,"GN_CARTBOOST"; bonus2 bSkillCooldown,"GN_MANDRAGORA",-1000; }
+19190:29321,{ autobonus3 "{ bonus bBaseAtk,40; bonus2 bAddSize,Size_All,15; }",1000,60000,"GN_CARTBOOST"; bonus2 bSkillCooldown,"GN_MANDRAGORA",-5000; }
+19191:29322,{ autobonus3 "{ bonus bBaseAtk,20; }",1000,60000,"ASC_EDP"; }
+19191:29323,{ autobonus3 "{ bonus bBaseAtk,30; }",1000,60000,"ASC_EDP"; bonus2 bAddSize,Size_All,5; }
+19191:29324,{ autobonus3 "{ bonus bBaseAtk,50; bonus bCritical,10; }",1000,60000,"ASC_EDP"; bonus2 bAddSize,Size_All,10; }
+19192:29325,{ autobonus3 "{ bonus bCritical,5; }",1000,60000,"SC_AUTOSHADOWSPELL"; }
+19192:29326,{ autobonus3 "{ bonus bCritical,10; }",1000,60000,"SC_AUTOSHADOWSPELL"; bonus2 bMagicAtkEle,Ele_Water,5; bonus2 bMagicAtkEle,Ele_Fire,5; bonus2 bMagicAtkEle,Ele_Wind,5; bonus2 bMagicAtkEle,Ele_Earth,5; }
+19192:29327,{ autobonus3 "{ bonus bCritical,20; bonus bMatk,30; }",1000,60000,"SC_AUTOSHADOWSPELL"; bonus2 bMagicAtkEle,Ele_Water,15; bonus2 bMagicAtkEle,Ele_Fire,15; bonus2 bMagicAtkEle,Ele_Wind,15; bonus2 bMagicAtkEle,Ele_Earth,15; }
+19193:2000,{ bonus2 bVariableCastrate,"HW_MAGICPOWER",50; }
```

Comment:
unintended?

Reply:
It has a description on bottom of description so I will change from varcast -> fixcast.